airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/AbstractJdbcDestination.kt

```diff
@@ -245,22 +244,7 @@ abstract class AbstractJdbcDestination<DestinationState : MinimumDestinationStat
         outputRecordCollector: Consumer<AirbyteMessage>
     ): SerializedAirbyteMessageConsumer? {
         val database = getDatabase(getDataSource(config))
-        // Short circuit for non-v2 destinations.
-        if (!isDestinationV2) {
-            return JdbcBufferedConsumerFactory.createAsync(
-                outputRecordCollector,
-                database,
-                sqlOperations,
-                namingResolver,
-                config,
-                catalog,
-                null,
-                NoopTyperDeduper(),
-            )
-        }
```

Comment:
yay!. no more non-dv2 code.

Reply:
there's probably dozens of these scattered everywhere, but yeah we can start wiping them out